feature_importance.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+
+import numpy as np
+import pickle
+import pandas as pd
+from sklearn.ensemble import RandomForestClassifier
+# use old version of scikit-learn==0.21.3
+import matplotlib.pyplot as plt
+from matplotlib.ticker import MultipleLocator, PercentFormatter
+
+# Feature order for model0 to model5
+feature_names = [
+    ['GWT', 'distance', 'Slope'],
+    ['GWT', 'distance', 'Slope','PGA'],
+    ['GWT', 'Elevation', 'distance', 'Slope'],
+    ['GWT', 'Elevation', 'distance', 'Slope', 'PGA'],
+    ['GWT', 'distance', 'Slope', 'PGA', 'qtncs', 'qtncs_std', 'Ic', 'Ic_std'],
+    ['GWT', 'Elevation', 'distance', 'Slope', 'PGA', 'qtncs', 'qtncs_std', 'Ic', 'Ic_std'],
+]
+
+
+
+def plot_importances(mdtype, num):
+    md_dir = './PRJ-2998/Model Usage/RF_' + mdtype + '_Model'+str(num)+'.pkl'
```

Comment:
I'd pass the model object instead of path. Separate input handling from the actual job of the function.